src/index.js

```diff
@@ -1,12 +1,54 @@
 import './styles/style.css';
-import UI from './modules/ui.js';
-import Todo from './modules/todo.js';
 
-document.addEventListener('DOMContentLoaded', UI.displayTodo);
+import updateTaskStatus from './models/task.js';
+import {
+  displayTask, addTask, deleteTask, lists,
+} from './models/all.js';
 
-// eslint-disable-next-line no-undef
+const form = document.getElementById('add-to-list');
+const btnClear = document.querySelector('.btn-clear');
 
-// eslint-disable-next-line no-undef, no-unused-vars
-const book = new Todo(desc, completed, id);
+const editTask = (task) => {
+  const tasks = JSON.parse(localStorage.getItem('tasks'));
+  const taskItem = task.children[0].children[1].name;
+  const taskIndex = tasks.findIndex((x) => x.description === taskItem);
+  const taskName = task.querySelector('#task-name').value;
+  tasks[taskIndex].description = taskName;
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
```

Comment:
- [ ] As noticed by the previous reviewer, the edit function does not take effect both on your page and in the local storage. Kindly look into this and fix it.